src/main/resources/application.properties

```diff
@@ -81,3 +81,7 @@ threads.core.pool.size: 200
 threads.queue.capacity: 7000
 threads.max.pool.size: 250
 scheduled.threadpool.size: 200
+
+logging.file.path=/eiffel-intelligence/eiffel-intelligence.log
```

Comment:
Better to use /tmp as the path as root path might not be accesible in different hosts